tests/tests.cc

```diff
@@ -27,16 +28,32 @@
 #include "mavlink_vehicles.hh"
 #include "tests.hh"
 
+void print_test_result(const char *test_name, bool result)
+{
+    if (result) {
+        std::cout << "[tests] [" << test_name << "] OK" << std::endl;
+    } else {
+        std::cout << "[tests] [" << test_name << "] FAIL" << std::endl;
+    }
+}
+
 int main()
 {
-    // tests::connection_test st;
-    // st.run();
+    std::shared_ptr<tests::common_test> test;
+
+    std::cout << "[tests] [Connection] Started" << std::endl;
+    test = std::make_shared<tests::connection_test>();
+    print_test_result("Connection", test->run(10));
 
-    // tests::conversion_test ct;
-    // ct.run();
+    std::cout << "[tests] [Conversion] Started" << std::endl;
+    test = std::make_shared<tests::conversion_test>();
+    print_test_result("Conversion", test->run());
 
-    tests::mission_test mt;
-    mt.run();
+    std::cout << "[tests] [Mission] Started" << std::endl;
+    test = std::make_shared<tests::mission_test>();
+    print_test_result("Mission", test->run(20));
```

Comment:
I agree that having this list would be very nice. I have implemented it [here](https://github.com/01org/mavlink-vehicles/pull/19/commits/a9bfec7bae0bca7a9b2b703530d146f3a9c7a048) as you suggested.

In order to implement it I had to take into consideration the following, though:
- The connection to the vehicle was being made into its constructor, and since we can't have multiple connections at the same time, it would be impossible to store a list with instantiated test_objects inside.

What I have done is to drop the connection part of the constructor and put it outside in a separate method - connection() - that needs to be called on demand into the run() methods of the tests in case they need to connect to the vehicle.